src/completion.rs

```diff
@@ -83,15 +87,28 @@ pub struct Stopping<'a> {
 }
 
 impl<'a> Stopping<'a> {
+    /// Only stop once the model generates end of text, or the sum of input tokens and generated
+    /// tokens has reached the model's context window size.
```

Comment:
Some models can't necessarily generate the entire context window, their max tokens may be less. But this would allow for maximum allowed tokens (not sure what to call it...)

Reply:
add more info the doc comment and call it 

`pub const NO_TOKEN_LIMIT`